checkov/sca_image/runner.py

```diff
@@ -73,8 +83,26 @@ async def execute_scan(
             logging.error(stderr.decode())
             return {}
 
-        # read and delete the report file
+        # read the report file
         scan_result: Dict[str, Any] = json.loads(output_path.read_text())
+
+        # upload results to cache
+        image_id_sha = ':'.join(['sha256', image_id])
```

Comment:
I suggest to use here also 'quote_plus'
`image_id_sha = quote_plus(f"sha256:{image_id}")`
(as you did in the image_scanner.py)

Reply:
Will modify to f-string, but I think the usage of `quote_plus` isn't needed and might lead to undesired behavior since the id is passed in the body and not the url itself so it won't be decoded